raiden/waiting.py

```diff
@@ -369,38 +372,51 @@ def wait_for_healthy(
     wait_for_network_state(raiden, node_address, NODE_NETWORK_REACHABLE, retry_timeout)
 
 
-def wait_for_transfer_success(
+class TransferWaitResult(Enum):
+    SECRET_REGISTERED_ONCHAIN = "secret registered onchain"
+    UNLOCKED = "unlocked"
+    UNLOCK_FAILED = "unlock_failed"
+
+
+def wait_for_transfer_result(
```

Comment:
Since you're renaming this function, please make add the idea that the node is *receiving* a transfer, and not sending it.